jax/api.py

```diff
@@ -406,25 +597,57 @@ def computation_maker(*args, **kwargs):
     else:
       out_parts_flat = tuple(flatten_axes(
           "xla_computation out_parts", out_tree(), out_parts))
-    c = xb.make_computation_builder('xla_computation_{}'.format(fun_name))
+    c = xb.make_computation_builder("xla_computation_{}".format(fun_name))
     xla_consts = map(partial(xb.constant, c), consts)
-    xla_args = xla._xla_callable_args(c, avals, tuple_args,
+    # As we cannot modify the closure-captured `tuple_args`, we work on a copy.
```

Comment:
Keep this more succinct?
```
should_tuple = tuple_args if tuple_args is not None else (len(avals) > 100)
```
and omit the comment?

Reply:
Done.